django/db/models/sql/query.py

```diff
@@ -491,14 +491,19 @@ def get_aggregation(self, using, added_aggregate_names):
             self.default_cols = False
             self.extra = {}
 
+        empty_aggregate_result = [
```

Comment:
Equally, as this is in `get_aggregation()` we can probably have:

```suggestion
        empty_result = [
```